raiden/raidenwebui/src/app/services/raiden.service.ts

```diff
@@ -168,6 +168,18 @@ export class RaidenService {
             .catch(this.handleError);
     }
 
+    public connectTokenNetwork(funds: number, tokenAddress: string): Observable<any> {
+        const data = {
+            'funds': funds
+        }
+        const headers = new Headers({ 'Content-Type': 'application/json' });
+        const options = new RequestOptions({ headers: headers });
+        return this.http.put(`${this.config.apiCall}/connection/${tokenAddress}`,
+        JSON.stringify(data), options).map((response) => {
```

Comment:
I think you can indent these lines under put, to visually indicate it is below `put`, and `.catch` on its own line too.